### DISCUSSION TITLE
esp 8266 tasmota con sensore ntc

### DISCUSSION THREAD

**Discussion by asbesto01** [indomus/forum#1510] [HUB: Homebridge & Apple HomeKit]
Ciao a tutti, 

alla centralina del mio impianto solare termico arrivano tre sonde NTC 10K.

Ho necessità di leggere le tre temperature per essere in grado di domotizzare l'impianto, e vorrei leggerle con un ESP8266 (nello specifico con un sonoff TH16) - in maniera da poter esporre le letture su Homekit.

Avete idea di come passare all'ESP8266 la temperatura di una sonda NTC 10K?

Grazie mille!

**Comment by snakuzzo** [comment]
Prendi spunto da [qui](https://indomus.it/guide/homebridge-mqtt-temperature-tasmota/)...
poi per l'esposizione verso Homekit...a te la scelta (se usare HA o altro)

**Reply by asbesto01** [reply]
> Prendi spunto da [qui](https://indomus.it/guide/homebridge-mqtt-temperature-tasmota/)...
> poi per l'esposizione verso Homekit...a te la scelta (se usare HA o altro)

Grazie mille, anche se non capisco come fare; nella guida che mi hai mandato usano un sensore AM2301, io invece voglio usare una sonda NTC 10k. Il primo ha due cavi, la seconda due!

come si fa?

grazie mille!
